packages/model/addon/-private/model.js

```diff
@@ -19,16 +20,38 @@ import {
   PromiseObject,
   recordDataFor,
   recordIdentifierFor,
-  RootState,
+  relationshipFromMeta,
+  typeForRelationshipMeta,
 } from '@ember-data/store/-private';
 
 import Errors from './errors';
-import {
-  relatedTypesDescriptor,
-  relationshipsByNameDescriptor,
-  relationshipsDescriptor,
-  relationshipsObjectDescriptor,
-} from './system/relationships/ext';
+import { computedMacroWithOptionalParams } from './util';
+
+const RecordMeta = new WeakMap();
+function getRecordMeta(record) {
+  let meta = RecordMeta.get(record);
+  if (meta === undefined) {
+    meta = Object.create(null);
+    RecordMeta.set(record, meta);
+    if (DEBUG) {
+      record.____recordMetaCache = meta;
+    }
```

Comment:
I don't remember the past conversations but I am not sure what the intention here was.

Reply:
It was to help with debugging/console access